userdata.tpl

```diff
@@ -2,9 +2,134 @@ MIME-Version: 1.0
 Content-Type: multipart/mixed; boundary="//"
 
 --//
-Content-Type: text/x-shellscript; charset="us-ascii"
-#!/bin/bash
+Content-Type: text/x-shellscript
+Content-Type: charset="us-ascii"
```

Comment:
AFAIK you are not allowed to have multiple `Content-Type` headers. For sure I see no advantage to it.